packages/cliqz-home-react/src/TopMessages.js

```diff
@@ -0,0 +1,44 @@
+import React, { Component } from 'react';
+import cliqz from './Cliqz';
+
+class TopMessages extends Component {
+
+  handleClose(id, handler) {    
+    cliqz.freshtab.dismissMessage(id, handler);
+    cliqz.storage.setState((prevState) => {
```

Comment:
here you use `cliqz` as a global object. 

in this architecture you should rather call an action passed on `props`